astropy/time/tests/test_corrs.py

```diff
@@ -44,3 +52,13 @@ def test_arrays(self):
         assert hval_arr[1]-hval2 < 1. * u.us
         assert bval_arr[0]-bval1 < 1. * u.us
         assert bval_arr[1]-bval2 < 1. * u.us
+
+    @remote_data
+    @pytest.mark.skipif('not HAS_JPLEPHEM')
+    def test_ephemerides(self):
```

Comment:
As above. This may mean finally biting the bullet and installing and figuring out TEMPO2.